### PR TITLE
LPS-147198 | Verify Remote App Entry can be configured at instance level

### PR DESCRIPTION
**Pull request by susanchen3**
Background
Create automated tests for Remote Apps Workflow

Test Plan
- Navigate to workflow configuration page
- Search for remote app entry
- Assert it is present

JIRA Ticket:
https://issues.liferay.com/browse/LPS-147198

### CODE REVIEW

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

File: modules/apps/frontend-css/frontend-css-test/src/testFunctional/tests/Cadmin.testcase [hunk 1]
```diff
@@ -44,6 +44,8 @@ definition {
 	@description = "LPS-146034. Verifies admin component displays it's default theme after applying a theme"
 	@priority = "5"
 	test CanDisplayDefaultTheme {
+		property portal.acceptance = "true";
+
```
Comment (susanchen3) [inline modules/apps/frontend-css/frontend-css-test/src/testFunctional/tests/Cadmin.testcase:48]: Realized I forgot to add the tests to acceptance... but both tests are [upstream](https://testray.liferay.com/home/-/testray/cases/1609092542) now :)

**Review by john-co — APPROVED** [review]
Looks good! Added an assertion to make certain remote apps is listed.

**Comment by john-co** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:forward

**Pull request closed without merging**